libs/SalesforceSDKCore/SalesforceSDKCore/Classes/RestAPI/SFNetwork.m

```diff
@@ -97,4 +141,53 @@ + (void)setSessionManager:(id<SFNetworkSessionManaging>)manager {
     kSFNetworkManager = manager;
 }
 
++ (NSDictionary *)sharedSessions {
```

Comment:
Create a class method for this instead of a `property`, once this becomes a `static` dictionary.